js/battle.js

```diff
@@ -4240,7 +4242,7 @@ var Battle = (function () {
 				poke.prevItem = item.name;
 				poke.prevItemEffect = '';
 				poke.removeVolatile('airballoon');
-
+				poke.addVolatile('unburden');
```

Comment:
"unburden" is the wrong name for this volatile, since it implies we know the pokemon has Unburden. I'd suggest "itemremoved" instead.